packages/instantsearch.js/src/widgets/looking-similar/looking-similar.tsx

```diff
@@ -0,0 +1,217 @@
+        templates,
+      });
+      return;
+    }
+
+    const headerComponent = (
+      templates.header
+        ? ({ classNames, recommendations }) => (
+            <TemplateComponent
+              {...renderState.templateProps}
+              templateKey="header"
+              rootTagName="fragment"
+              data={{ cssClasses: classNames, recommendations }}
+            />
+          )
+        : undefined
+    ) as LookingSimilarUiProps<Hit>['headerComponent'];
+
+    const itemComponent = (
+      templates.item
+        ? ({ item }) => {
+            return (
+              <TemplateComponent
+                {...renderState.templateProps}
+                templateKey="item"
+                rootTagName="fragment"
+                data={item}
+              />
+            );
+          }
+        : undefined
+    ) as LookingSimilarUiProps<Hit>['itemComponent'];
+
+    const fallbackComponent = (
+      templates.empty
+        ? () => (
+            <TemplateComponent
+              {...renderState.templateProps}
+              templateKey="empty"
+              rootTagName="fragment"
+              data={results}
+            />
+          )
+        : undefined
+    ) as LookingSimilarUiProps<Hit>['fallbackComponent'];
+
+    render(
+      <LookingSimilar
+        items={receivedRecommendations}
+        headerComponent={headerComponent}
+        itemComponent={itemComponent}
+        sendEvent={() => {}}
+        classNames={cssClasses}
+        fallbackComponent={fallbackComponent}
+        status={instantSearchInstance.status}
+      />,
+      containerNode
+    );
+  };
+
+export type LookingSimilarCSSClasses = Partial<RecommendClassNames>;
+
+export type LookingSimilarTemplates = Partial<{
+  /**
+   * Template to use when there are no results.
+   */
+  empty: Template<RecommendResultItem>;
+
+  /**
+   * Template to use for the header of the widget.
+   */
+  header: Template<
+    Pick<
+      Parameters<NonNullable<LookingSimilarUiProps<Hit>['headerComponent']>>[0],
+      'recommendations'
+    > & { cssClasses: RecommendClassNames }
+  >;
+
+  /**
+   * Template to use for each result. This template will receive an object containing a single record.
+   */
+  item: Template<Hit>;
+}>;
+
+type LookingSimilarWidgetParams = {
+  /**
+   * CSS Selector or HTMLElement to insert the widget.
+   */
+  container: string | HTMLElement;
+
+  /**
+   * Templates to use for the widget.
+   */
+  templates?: LookingSimilarTemplates;
+
+  /**
+   * CSS classes to add.
+   */
+  cssClasses?: LookingSimilarCSSClasses;
+};
+
+export type LookingSimilarWidget = WidgetFactory<
+  LookingSimilarWidgetDescription & {
+    $$widgetType: 'ais.lookingSimilar';
+  },
+  LookingSimilarConnectorParams,
+  LookingSimilarWidgetParams
+>;
+
+const lookingSimilar: LookingSimilarWidget = function lookingSimilar(
+  widgetParams
+) {
+  const {
+    container,
+    objectIDs,
+    maxRecommendations,
+    queryParameters,
```

Comment:
It should forward `fallbackParameters`:

```suggestion
    queryParameters,
    fallbackParameters,
```